actors/migration/nv3/reward.go

```diff
@@ -5,11 +5,36 @@ import (
 
 	cid "github.com/ipfs/go-cid"
 	cbor "github.com/ipfs/go-ipld-cbor"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/specs-actors/actors/builtin/reward"
 )
 
 type rewardMigrator struct {
 }
 
 func (m *rewardMigrator) MigrateState(ctx context.Context, store cbor.IpldStore, head cid.Cid) (cid.Cid, error) {
-	return head, nil
+	var st reward.State
+	if err := store.Get(ctx, head, &st); err != nil {
+		return cid.Undef, err
+	}
+
+	// The baseline function initial value and growth rate are changed.
+	// As an approximation to working out what the baseline value would be at the migration epoch
+	// had the baseline parameters been this way all along, this just sets the immediately value to the
+	// new (higher) initial value, essentially restarting its calculation.
+	// The baseline and realized cumsums, and effective network time, are not changed.
+	// This boils down to a step change in the baseline function, as if it had been defined piecewise.
+	// This will be a bit annoying for external analytical calculations of the baseline function.
+	newBaselineInitialValue := reward.BaselineInitialValueV3
+
+	if st.ThisEpochBaselinePower.GreaterThan(newBaselineInitialValue) {
+		return cid.Undef, xerrors.Errorf("unexpected baseline power %v higher than new initial value %v",
```

Comment:
Do we really want to crash the migration for this reason?  What would we do in this case, change the new baseline function?  Does anything go immediately wrong if this value falls below the current baseline.  I am very suspicious that no code will break.

Reply:
Well, if this doesn't hold, we must have a serious misunderstanding about how the baseline function works. But ok, perhaps a sanity check can go in the state check later instead.